<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Biomeng321Lab2</class>
 <widget class="QWidget" name="Biomeng321Lab2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biomeng321 Lab2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Solution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Strain Analysis</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Deformation Gradient Tensor</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>F</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidgetDeformationGradient">
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Invariants</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>I1:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditInvariant1"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>I2:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditInvariant2"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>I3:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditInvariant3"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Right Cauchy-Green Deformation Tensor</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidgetRightCauchyGreenDeformation">
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Green-Lagrange Strain Tensor</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>E</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>(ref)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>=</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidgetGreenLagrangeStrainReference">
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>E</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>(fib)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>=</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidgetGreenLagrangeStrainFibre">
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_10">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Stress Analysis</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Hydrostatic Pressure</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Value:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditHydrostaticPressure"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Second Piola-Kirchhoff Stress Tensor</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>T</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>(ref)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>=</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidgetSecondPiolaKirchoffStressReference">
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_19">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>T</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>(fib)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>=</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidgetSecondPiolaKirchoffStressFibre">
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Cauchy Stress Tensor</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="2">
             <widget class="QTableWidget" name="tableWidgetCauchyStress">
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>sigma</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
